app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -119,9 +129,36 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
 
     private fun handleChooseFolder(folderId: Long, folderDisplayName: String) {
         selectedFolderId = folderId
+        selectedFolderDisplayName = folderDisplayName
         unreadFolder.summary = folderDisplayName
     }
 
+    private fun loadSavedState(savedInstanceState: Bundle?) {
```

Comment:
I'd rename this method to `restoreInstanceState()`. This is more consistent with the Android terminology.

Also, please move this method to either right before or right after `onSaveInstanceState()`. Code to save and restore state should be close to each other.